guacamole/src/main/frontend/src/translations/ca.json

```diff
@@ -163,6 +183,80 @@
+        "HELP_JSON_MORE_DETAILS"            : "The connection group ID that the connection should be imported into may be directly specified with a \"parentIdentifier\" field, or the path to the parent group may be specified using a \"group\" field as shown below. An array of user and user group identifiers to grant access to may be specified per connection.",
+        "HELP_EXISTING_CONNECTION_MODE"     : "Entirely replace/update existing connections if their names and parent connection groups match the values in the provided file. If unchecked, attempting to import a connection with the same name and parent connection group of an existing connection will be considered an error.",
+        "HELP_EXISTING_PERMISSION_MODE"     : "Fully reset the permissions granted for all connections in the provided file to the permissions specified in that file. If no permissions are specified, all relevant connection permissions will be revoked. If unchecked, existing permissions are preserved, and any permissions specified in the file will be added.",
+        "HELP_SEMICOLON_FOOTNOTE"           : "Si cal, els punt i coma es poden escapar utilitzant una contrabarra, per exemple. \"first\\\\;last\"",
+        "HELP_UPLOAD_DROP_TITLE"            : "Arrossegueu un fitxer aquí",
+        "HELP_UPLOAD_FILE_TYPES"            : "CSV, JSON, o YAML",
+        "HELP_YAML_DESCRIPTION"             : "Un fitxer YAML d'importació de connexions és una llista d'objectes de connexió amb exactament la mateixa estructura que el format JSON.",
+        "HELP_YAML_EXAMPLE"                 : "---\n  - name: conn1\n    protocol: vnc\n    parameters:\n      username: alice\n      password: pass1\n      hostname: conn1.web.com\n    group: ROOT\n    users:\n      - guac user 1\n      - guac user 2\n    groups:\n    - Connection 1 Users\n    attributes:\n      guacd-encryption: none\n  - name: conn2\n    protocol: rdp\n    parameters:\n      username: bob\n      password: pass2\n      hostname: conn2.web.com\n    group: ROOT/Parent Group\n    users:\n      - guac user 1\n    attributes:\n      guacd-encryption: none\n  - name: conn3\n    protocol: ssh\n    parameters:\n      username: carol\n      password: pass3\n      hostname: conn3.web.com\n    group: ROOT/Parent Group/Child Group\n    users:\n      - guac user 2\n      - guac user 3\n  - name: conn4\n    protocol: kubernetes",
```

Comment:
Several of the items in this section appear to have not been translated. Items that you are not translating should not be included in this file - it will automatically fall back to the English translation.